tests/func/test_live.py

```diff
@@ -81,19 +89,11 @@ def make(summary=True, html=True, live=None, live_no_cache=None):
     yield make
 
 
-@pytest.mark.parametrize("report", (True, False))
+@pytest.mark.parametrize("html", (True, False))
 @pytest.mark.parametrize("summary", (True, False))
-def test_export_config_tmp(tmp_dir, dvc, mocker, summary, report):
```

Comment:
remove the `test_export_config_tmp` test - now that `live` is online `test_export_config` tests the same things